scancode-toolkit.ABOUT

```diff
@@ -5,4 +5,5 @@ copyright: Copyright (c) nexB Inc. and others.
 homepage_url: https://www.aboutcode.org/
```

Comment:
Since we are doing a cleanup, we should probably delete this file, since this does not add any data to the package manifests already present here for scancode-toolkit.

Reply:
For the purpose of the ABOUT file, I think we still want to keep this file.